tests/test_scripts/__snapshots__/gengraphql/meta.graphql

```diff
@@ -1,4 +1,43 @@
 # metamodel_version: 1.7.0
+enum CordialnessEnum
+  {
+    heartfelt
+    hateful
+    indifferent
+  }
+
+enum DiagnosisType
+  {
+    TODO
+  }
+
+enum EmploymentEventType
+  {
+    HIRE
+    FIRE
+    PROMOTION
+    TRANSFER
+  }
+
+enum FamilialRelationshipType
+  {
+    SIBLING_OF
+    PARENT_OF
+    CHILD_OF
+  }
+
+enum LifeStatusEnum
+  {
+    LIVING
+    DEAD
+    UNKNOWN
+  }
+
+enum OtherCodes
+  {
+    a__SPACE__b
```

Comment:
i feel like space should pretty much always be transformed to `_` ;p

Reply:
But then there conversion can only go in one direction, because once you find a single `_`, you cannot tell if it was originally a `_` or a ` `.

Reply:
i think being able to invert the transformation is good (and ideal), but since we don't have an inverter in hand, i think if we go by convention this is what's done in other generators? i also don't have strong feelings here but i think graphqlgen might be the only generator that does this. am aware i was the one who brought up invertible transformations so i am not trying to mess with you and really really don't want to keep adding stuff for you to do here!!!!

Reply:
I have decided to change the code according @sneakers-the-rat opinion to hopefully get this PR finally merged. Since I do not really have a strong stake at inverting the transformation, I will go for the well-established practice.